spec/integration/support/inspect_changed_managed_files_examples.rb

```diff
@@ -55,12 +62,7 @@
         actual_managed_files_list.grep(/^#{element}.+/).any? || element == "."
       }
 
-      expected_managed_files = []
-      expected_files_list.split("\n").each do |e|
-        if e.start_with?("  * ") && !e.include?("(deleted)") && !e.include?("link_path")
-          expected_managed_files << ".#{e.split[1]}"
-        end
-      end
+      expected_managed_files = description.changed_managed_files.files.select { |f| f.file? }.map(&:name)
```

Comment:
Line is too long. [105/100]<br>Pass `&:file?` as an argument to `select` instead of a block.